_translator-files/po/es/Software-Manual.po

```diff
@@ -251,27 +249,23 @@ msgstr "<figure><img src=\"{% include img/es-screenshots/connection-setup-window
 msgid "The Connection Setup window shows a list of available Servers together with their \"distance\" (in terms of ping time), the number of occupants and the maximum number supported, and the Server's given location.  Permanent Servers (those that have been listed for longer than 24 hours) are shown in bold."
-msgstr "La ventana de Configuración de Conexión muestra una lista de servidores disponibles junto con el número de usuarios conectados y el número máximo de los mismos soportado. Los operadores de servidores los registran en listas (mayormente definidas por género, aunque algunas pueden referirse a ubicación o a todos los géneros). Utiliza el menú desplegable de Lista para elegir un género, haz clic sobre el servidor al que te quieres unir y otra vez sobre el botón de Conectar para conectarte. También puedes hacer doble clic sobre el nombre del servidor. Los servidores permanentes (aquellos que llevan más de 24 horas en la lista) se muestran en negrita."
+msgstr "La ventana de Configuración de Conexión muestra una lista de Servidores disponibles con su \"distancia\" (en términos de tiempo ping), junto con el número de usuarios conectados y el número máximo de los mismos soportado, así como la ubicación del Servidor. Los Servidores permanentes (aquellos que llevan más de 24 horas en la lista) se muestran en negrita."
 
 #. type: Plain text
 #: ../wiki/en/Software-Manual.md:88
 msgid "Click on the Server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name."
-msgstr ""
+msgstr "Haz clic en el Servidor al que te quieras unir y dale al botón de Conectar para realizar la conexión. También puedes hacer doble-clic en el nombre del servidor."
 
 #. type: Plain text
 #: ../wiki/en/Software-Manual.md:92
 msgid "Server operators register their servers with Directories. These are either genre-specific or for any genre, or additional [custom entries](#custom-directories). Use the Directory dropdown to select a genre.  You can filter the list by server name or location. To list only occupied Servers, enter a \"#\" character.  If you choose \"Show All Musicians\" the current occupants will be shown."
-msgstr ""
+msgstr "Los operadores de servidores registran sus servidores en Directorios. Éstos pueden ser para un género específico o para cualquiera, o ser [personalizados](#directorios personalizados). Utiliza el menú desplegable de Directorios para seleccionar un género.  Puedes filtrar la lista por nombre de servidor o por ubicación. Para mostrar solo los Servidores ocupados, introduce el caracter \"#\".  Si eliges \"Mostrar Todos los Músicos\", se mostrarán los usuarios actuales conectados."
```

Comment:
Probably #directorios personalizados will be a broken link. It'll probably be "#directorios-personalizados"

Reply:
Yes, you're right.

Reply:
Corrected.